packages/gateway/ava.config.js

```diff
@@ -4,5 +4,6 @@ export default {
   },
   files: ['test/*.spec.js'],
   timeout: '5m',
+  concurrency: 1,
```

Comment:
Running the test cases in parallel makes cache test flakey, likely because other Miniflare instances run and they reset the cache 🤷🏼  Needed to make concurrency 1 to avoid it